goal_number: 7# goal_1: 1#goal_1: 1 #number : region#goal_2: 2#goal_3: 1#goal_4: 9#goal_5: 1goal_1: 1 #number : regiongoal_2: 2goal_3: 10 goal_4: 5goal_5: 7# goal_1: 5 #number : region# goal_2: 4# goal_3: 5# goal_4: 1# goal_5: 1goal_6: 4goal_7: 11goal_8: 10goal_9: 8goal_10: 9goal_11: 1head_angle: -90head_region: 2